salt/modules/cmdmod.py

```diff
@@ -22,6 +22,7 @@
 import base64
 import re
 import tempfile
+from distutils.spawn import find_executable
```

Comment:
Please import this as a private function (i.e. `from distutils.spawn import find_executable as _find_executable`). Importing a function into the global namespace of the module will both a) cause it to be picked up by the loader (and thus end up in the `__salt__` dunder dictionary), and b) add it to the docs when they are built using Sphinx.

Reply:
Is there a reason we shouldn't just use `salt.utils.path.which_bin`?

Reply:
Thanks for the tip. I hadn't noticed it.